react-native/react/update/index.render.desktop.js

```diff
@@ -0,0 +1,172 @@
+import React, {Component} from '../base-react'
+import {Checkbox, FlatButton} from 'material-ui'
+import Snackbar from 'material-ui/lib/snackbar'
+import commonStyles, {colors} from '../styles/common'
+import Header from '../common-adapters/header'
+import {clipboard} from 'electron'
+import resolveAssets from '../../../desktop/resolve-assets'
+import marked from 'marked'
+import {resizeWindow} from '../native/remote-component-helper'
+
+export default class UpdateRender extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+    }
+  }
+
+  componentDidMount () {
+    resizeWindow()
+  }
+
+  componentDidUpdate () {
+    resizeWindow()
```

Comment:
It could be part of remote component itself and a prop flag

On Thursday, December 17, 2015, Marco Munizaga <notifications@github.com>
wrote:

> In react-native/react/update/index.render.desktop.js
> <https://github.com/keybase/client/pull/1544#discussion_r47978219>:
>
> > +import {resizeWindow} from '../native/remote-component-helper'
> > +
> > +export default class UpdateRender extends Component {
> > +  constructor (props) {
> > +    super(props)
> > +
> > +    this.state = {
> > +    }
> > +  }
> > +
> > +  componentDidMount () {
> > +    resizeWindow()
> > +  }
> > +
> > +  componentDidUpdate () {
> > +    resizeWindow()
>
> I'm not sold on this, because this kinda assumes this component will be in
> a remote component. But it shouldn't know that at all
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1544/files#r47978219>.
>